### PR TITLE
Added CentralScene CommandClass

### PR DESCRIPTION
**Pull request by bluewalk**
You can ignore the removal of NLog of course as the original lib uses this :-)

### CODE REVIEW

**Comment by bluewalk** [comment]
I've resolved the issues and changed to UtcNow as requested :-)

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ﻿# Z-Wave Home Automation library for .NET
 
+## About this fork
+I've forked this to remove NLog as the project I am using this library in doesn't use NLog thus adding more libraries seem useless.
+This fork also contains WIP's for adding new commandclasses to the library when I stumble upon one that's missing.
+
```
Comment (genemars) [inline README.md:6]: If the intention is to merge into the main repo, this note should be here. Also removal of NLog is not a good thing. There are other projects already using this library that also relay on NLog.
So adding *CentralScene CommandClass* is ok as long the PR doesn't involve structural changes to the ZWaveLib project that have not been discussed.

**Review by genemars — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bluewalk) [reply README.md:6]: Agreed, the removal of NLog is more a thing that I use in my own projects as it has a different way of logging.
Furthermore this is basically my first pull request ever so I don't know if you can ignore the other changes beside the ones regarding the CentralScene ones?

Comment (genemars) [reply README.md:6]: @bluewalk I think you create a new branch. Just add the *CentralScene* command class there and submit the PR. If you need other specific customizations that are not supposed to be merged in the master repo you can create other branches and sync with master whenever there are updates from the upstream.

**Comment by bluewalk** [comment]
Will do as suggested. Will create new PR when ready!

**Comment by kjetilpp** [comment]
Any update?

**Comment by Bounz** [comment]
Hi, @kjetilpp.
This was merged with PR #23.

**Pull request closed without merging**